.github/workflows/kuksa_databroker_build.yml

```diff
@@ -301,28 +272,22 @@ jobs:
             target/
           key: databroker-bom-${{ hashFiles('**/Cargo.lock') }}
 
-      - name: Install prerequisites
-        working-directory: ${{github.workspace}}/createbom
+      # Follows the pattern from
+      # https://github.com/eclipse/dash-licenses?tab=readme-ov-file#example-rustcargo
+      - name: "Using cargo to create Dash input"
+        working-directory: ${{github.workspace}}/
+        # target all is not really needed, and will also return i.e. wasm deps, however
+        # better safe than sorry, the alternative would be running this for each currently
+        # built target  and combining the lists, but that would need adapting, when
+        # adding targets, or also when i.e. switching between MUSL/glibc. So this is safer
         run: |
-         which cargo-license || cargo install cargo-license
-      - name: License check and Dash output generation
-        working-directory: ${{github.workspace}}/createbom
-        run: |
-         python3 createbom.py --dash ${{github.workspace}}/dash-databroker ../databroker
+          cargo tree -e normal --prefix none --no-dedupe -p databroker --target all --all-features  > ${{github.workspace}}/cargodeps
+          cat ${{github.workspace}}/cargodeps | sort -u  \
+          | grep -v '^[[:space:]]*$' | grep -v kuksa | grep -v databroker  \
+          | sed -E 's|([^ ]+) v([^ ]+).*|crate/cratesio/-/\1/\2|' \
+          > ${{github.workspace}}/dash-databroker-deps
+
       - name: Dash license check
-        uses: eclipse-kuksa/kuksa-actions/check-dash@3
+        uses: eclipse-kuksa/kuksa-actions/check-dash@2
```

Comment:
Why do you go back to v2? I guess that will cause problems if you run https://github.com/eclipse-kuksa/kuksa-databroker/blob/main/.github/workflows/create_draft_release.yml (as then the Dash artifact has the same name for both databroker and databroker-cli which gives a conflict)

Reply:
copy & paste fail. Should be fixed